BUILD.md

```diff
@@ -133,9 +133,12 @@ Options can be enabled by specifiying `-DKokkos_ENABLE_X`.
 * Kokkos_ENABLE_COMPILER_WARNINGS
     * Whether to print all compiler warnings
     * BOOL Default: OFF
+* Kokkos_ENABLE_CUDA_CONSTEXPR
+    * Whether to activate experimental relaxed constexpr functions
+    * BOOL Default: ON
```

Comment:
Might want to change the default according to https://github.com/kokkos/kokkos/issues/2411#issuecomment-537261743